doc_user/UserManual.adoc

```diff
@@ -801,6 +801,288 @@ image:image251.png[success]
 
 NOTE: This button can be enabled or disabled to control movements deletions. Ask the Administrator or check the Administrator's Guide for more information.
 
+=== 4.4 Inventory ([.underline]##I##nventory)
+
```

Comment:
I would add a wide introduction about the new inventory feature, something like this:

```
The Inventory feature in the Open Hospital application enables users to manage stock levels within the hospital's Main Store. This functionality allows for the accurate tracking and adjustment of inventory, ensuring that stock levels are correctly maintained and recorded.

The application supports two types of inventory management:
- Main Store Inventory: this is the primary inventory management system for the hospital's central storage area. The details of this system are covered in this chapter.
- Ward Inventory: this feature is used to manage stock levels within individual wards of the hospital. The specifics of Ward Inventory management are explained in chapter 4.5.

```